README.md

```diff
@@ -83,6 +83,19 @@ of options:
  * [Libvirt][libvirt]
  * [Cockpit][cockpit]
 
+## Submitting patches
+
+When sending patches to the project, the submitter is required to certify that
+they have the legal right to submit the code. This is achieved by adding a line
+
+    Signed-off-by: Real Name <email@address.com>
```

Comment:
It is about setting up the project with a strong long term legal foundation for contributions, because we can't reliably predict the future. While there are certainly plenty of projects which don't do this, it is commonplace best practice for projects with a large & diverse contributor base (or which aim to have such in the future). It gives an explicit record of compliance for all contributions, which is particularly useful when some users are contributing code that was actually written by someone else, as this may not be reliably recorded in git author metadata